frameworks/elastic/docs/tls.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt:
+title: Configuring TLS
+menuWeight: 25
```

Comment:
Is this the correct menu position. @triclambert who do we reach out to w.r.t documentation issues now?